cli/local/docker_spec.go

```diff
@@ -183,7 +188,25 @@ func deployPythonContainer(api *spec.API, awsClient *aws.Client) error {
 
 	err = docker.MustDockerClient().ContainerStart(context.Background(), containerInfo.ID, dockertypes.ContainerStartOptions{})
 	if err != nil {
-		return errors.Wrap(err, api.Identify())
+		if api.Compute.GPU > 0 && strings.Contains(err.Error(), "could not select device driver \"\" with capabilities: [[gpu]]") {
```

Comment:
I'm a little worried that checking the string so precisely could be brittle, what do you think about shortening the string match requirement?